db/migrate/20230411085954_restore_coronation_topical_event.rb

```diff
@@ -0,0 +1,5 @@
+class RestoreCoronationTopicalEvent < ActiveRecord::Migration[7.0]
+  def change
+    TopicalEvent.unscoped.find_by(slug: "coronation").update(state: "current")
```

Comment:
Ooh before you merge, I imagine this migration is at risk of failing for a user without the expected record which can be a pain for devs

Reply:
Actually since it's run manually, i guess there's no need for the safe navigator